tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -19,6 +19,7 @@ import (
 
 var _ = Describe("Create machinepool",
 	labels.Feature.Machinepool,
+	labels.Runtime.Day2,
```

Comment:
Runtime should remain on the test case itself: https://github.com/openshift/rosa/blob/master/tests/README.md#labels

Reply:
Updated